build.gradle

```diff
@@ -156,6 +158,23 @@ subprojects {
         }
     }
 
+    // Specific setup for a Travis build,
+    // which prevents appearing of irrelevant messages in build logs.
```

Comment:
Please change `irrelevant` to `warning`. Please also add the following line:

`It is expected that warnings are viewed and analyzed in the local build logs.`